udata_gouvfr/views/territories.py

```diff
@@ -100,8 +100,11 @@ def render_territory(territory):
     # Retrieve the present territory if not presently valid.
     present_territory = None
     if not is_present_territory:
-        present_territory = GeoZone.objects.valid_at(date.today()).get(
-            level=territory.level, ancestors__contains=territory.id)
+        try:
+            present_territory = GeoZone.objects.valid_at(date.today()).get(
+                level=territory.level, ancestors__contains=territory.id)
+        except GeoZone.DoesNotExist:
```

Comment:
Not sure if we should make sure that it doesn't happen instead of catching the exception.

From what I understand, if we follow this logic, we should also catch `MultipleOjectsReturned` (see https://mongoengine-odm.readthedocs.io/apireference.html#mongoengine.queryset.QuerySet.get) as it happens in the other sentry error as well: https://sentry.data.gouv.fr/organizations/sentry/issues/12090/?project=5&query=is%3Aunresolved.